content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -31,13 +31,22 @@ When you configure redirection to a waypoint, traffic will be forwarded by ztunn
 
 ## Deploy a waypoint proxy
 
-Waypoint proxies are deployed declaratively using Kubernetes Gateway resources. You can use istioctl experimental subcommands to generate, apply or list these resources.
+Waypoint proxies are deployed using standard Kubernetes Gateway resources. 
+
+You need to install the [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) CRDs, which don’t come installed by default on most Kubernetes clusters:
+
+{{< text syntax=bash snip_id=install_k8s_gateway_api >}}
+$ kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
+  { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref=v1.1.0" | kubectl apply -f -; }
+{{< /text >}}
```

Comment:
There's a boilerplate for this

Reply:
@craigbox there is (and I see a few other spots we aren't using it), but it includes a step that

- We didn't include here before
- Isn't necessary for _ambient_ specifically
- Has a WARN text


e.g. https://preliminary.istio.io/latest/docs/tasks/traffic-management/tcp-traffic-shifting/

Should we create a new boilerplate, tweak the existing one to remove e.g. `PILOT_ENABLE_ALPHA_GATEWAY_API`, or just stick with this (which was copied from the existing doc) for now?

Reply:
Actually, it seems like we do not even need the `experimental` version of the CRDs anyway, for ambient waypoints.

Lemme test and confirm and if so I will leave this non-boilerplated - we should _not_ need to mention this outside of the actual waypoint doc (what this PR does) so it maybe shouldn't be boilerplated for ambient docs.

Reply:
Aight @craigbox updated this to use the non-experimental version.

PTAL. I think I would prefer if we just link to this doc in Ambient docs when we need to mention waypoints, rather than stick the boilerplate in a bunch of places.